network-arbitrary.cabal

```diff
@@ -1,5 +1,5 @@
 name:                network-arbitrary
-version:             0.5.0.0
+version:             0.5.0.1
```

Comment:
We don't bump version numbers in commits outside of the release branch and follow git flow.  You can drop this change.  Also, I doubt this would be the right version.  If you're curious, see https://pvp.haskell.org/ for details.

Reply:
reverted.